github3/orgs.py

```diff
@@ -269,23 +251,30 @@ class Organization(BaseAccount):
     members_roles = frozenset(['all', 'admin', 'member'])
 
     def _update_attributes(self, org):
-        super(Organization, self)._update_attributes(org)
-        self.type = self.type or 'Organization'
+        #: URL of the avatar at gravatar
+        self.avatar_url = org['avatar_url']
+
+        # Set the type of object (this doens't come through API data)
+        self.type = 'Organization'
+
+        self.url = self._api = org['url']
 
-        #: Events url (not a template)
-        self.events_url = self._get_attribute(org, 'events_url')
-        #: Number of private repositories.
-        self.private_repos = self._get_attribute(org, 'private_repos')
+        #: Unique ID of the account
+        self.id = org['id']
+
+        #: Name of the organization
+        self.login = org['login']
 
         #: Public Members URL Template. Expands with ``member``
-        self.public_members_urlt = self._class_attribute(
-            org,
-            'public_members_url',
-            URITemplate
-        )
+        self.public_members_urlt = URITemplate(org['public_members_url'])
 
-        #: Repositories url (not a template)
-        self.repos_url = self._get_attribute(org, 'repos_url')
+        #: Various urls (not templates)
+        for urltype in ('avatar_url', 'events_url', 'issues_url',
+                        'repos_url'):
+            self.__setattr__(urltype, org[urltype])
```

Comment:
Have I done something to make `setattr(self, urltype, org[urltype])` unusable?

Reply:
Nope, I just forgot that particular syntax. I have no excuse 😄